hail/python/hail/methods/qc.py

```diff
@@ -491,13 +507,335 @@ def n_discordant(counter):
+                                                     cds_start=hl.tint32,
+                                                     codons=hl.tstr,
+                                                     consequence_terms=hl.tarray(hl.tstr),
+                                                     distance=hl.tint32,
+                                                     domains=hl.tarray(hl.tstruct(db=hl.tstr,
+                                                                                  name=hl.tstr)),
+                                                     exon=hl.tstr,
+                                                     gene_id=hl.tstr,
+                                                     gene_pheno=hl.tint32,
+                                                     gene_symbol=hl.tstr,
+                                                     gene_symbol_source=hl.tstr,
+                                                     hgnc_id=hl.tstr,
+                                                     hgvsc=hl.tstr,
+                                                     hgvsp=hl.tstr,
+                                                     hgvs_offset=hl.tint32,
+                                                     impact=hl.tstr,
+                                                     intron=hl.tstr,
+                                                     lof=hl.tstr,
+                                                     lof_flags=hl.tstr,
+                                                     lof_filter=hl.tstr,
+                                                     lof_info=hl.tstr,
+                                                     minimised=hl.tint32,
+                                                     polyphen_prediction=hl.tstr,
+                                                     polyphen_score=hl.tfloat,
+                                                     protein_end=hl.tint32,
+                                                     protein_start=hl.tint32,
+                                                     protein_id=hl.tstr,
+                                                     sift_prediction=hl.tstr,
+                                                     sift_score=hl.tfloat,
+                                                     strand=hl.tint32,
+                                                     swissprot=hl.tstr,
+                                                     transcript_id=hl.tstr,
+                                                     trembl=hl.tstr,
+                                                     uniparc=hl.tstr,
+                                                     variant_allele=hl.tstr)),
+        variant_class=hl.tstr)
+
+    @staticmethod
+    def from_dict(config: Dict[str, Any]):
+        return VEPConfig(
+            config['data_bucket'],
+            config['regions'],
+            config['image'],
+            config['data_mount'],
+            config['env'],
+            config['vep_json_typ'],
```

Comment:
Do we have a standard way to serialize types? If not, we should have one, but you'll need to do some post-processing here.